<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search</class>
 <widget class="QWidget" name="search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <property name="styleSheet">
   <string notr="true">search
{
	background-color:#0a0612;
}
QPushButton
{
	background-color:#8a02b2;
	color:#1f1137;
	border-radius:20px;
}
QPushButton:hover
{
	background-color:#341d5c;
	color: #ffffff;
}
QLabel
{
	color:#ad03de;
}
QLineEdit
{
	background-color:#673ab7;
	color:#ffffff;
	border-radius:16px;
}
QLineEdit:hover
{
	background-color:#5d34a5;
}
QPlainTextEdit
{
	color:#ffffff;
	background-color:#3e236e;
	border-radius:12px;
}
QCheckBox
{
	color:#ad03de;
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>861</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="le_search">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_search">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_refresh">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>140</y>
     <width>781</width>
     <height>632</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QPlainTextEdit" name="pte_user5">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="pte_user2">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="pte_user1">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_follow1">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="pte_user3">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPlainTextEdit" name="pte_user4">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPlainTextEdit" name="pte_user6">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_follow2">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn_follow3">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn_follow4">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_follow5">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn_follow6">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>640</y>
     <width>61</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_up">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_down">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>690</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
